app/services/manage_courses_api_service.rb

```diff
@@ -0,0 +1,31 @@
+require 'net/http'
+
+class ManageCoursesAPIService
+  @api_base_url = ENV['MANAGE_API_BASE_URL'] || 'https://www.example.com/api'
+  @api_key = ENV['MANAGE_API_KEY'] || 'Bearer 12345'
```

Comment:
By convention, these should be moved and managed in an initializer in `config/initializers/`.